src/guides/v2.3/install-gde/system-requirements-tech.md

```diff
@@ -36,23 +36,22 @@ Upgrading the Magento applications and extensions you obtain from Magento Market
 
 ## Database
 
-MySQL 5.6, 5.7
+*  MySQL 5.7 or on-premise installations
+*  MariaDB 10.2 for {{site.data.var.ece}} projects
 
-Magento is also compatible with MySQL NDB Cluster 7.4.*, MariaDB 10.0, 10.1, 10.2, Percona 5.7, and other binary-compatible MySQL technologies.
+Magento is also compatible with, but has not been tested and is not recommended, with MySQL 5.6, MariaDB 10.1, and Percona 5.7.
 
 {:.bs-callout-info}
 Magento only uses MySQL features compatible with MariaDB. MariaDB may not be compatible with all MySQL features, however, so be sure to research compatibility issues before using a feature in your Magento module.
 
 ## PHP
 
+*  PHP ~7.3.0
+
```

Comment:
what about 7.4 (we have it on the Cloud versions table, but wanted to check to see if we have a mistake.)

Reply:
Product Management requested that we leave PHP ~7.3.0 only for Magento 2.3.x.

Reply:
7.4 is for 2.4 -- never mind.